server/athenian/api/serialization.py

```diff
@@ -251,6 +252,8 @@ def serialize(klass, obj):
             return obj.tolist()
         if isinstance(obj, (bytes, bytearray)):
             return obj.decode()
+        if isinstance(obj, enum.Enum):
+            return obj.value
```

Comment:
Our models cannot support enums due to the planned future native code, we should work with strings here. Take `.value` in the endpoint code pls.